pkg/disk/partition_table_test.go

```diff
@@ -1221,3 +1120,1013 @@ func TestAddPartitionsForBootMode(t *testing.T) {
+							FSTabOptions: "defaults,uid=0,gid=0,umask=077,shortname=winnt",
+							FSTabFreq:    0,
+							FSTabPassNo:  2,
+						},
+					},
+					// root is aligned to the end but not reindexed
+					{
+						Start:    222 * datasizes.MiB,
+						Size:     3*datasizes.GiB + datasizes.MiB - (disk.DefaultSectorSize + (128 * 128)), // grows by 1 grain size (1 MiB) minus the unaligned size of the header to fit the gpt footer
+						Type:     disk.FilesystemDataGUID,
+						UUID:     "a178892e-e285-4ce1-9114-55780875d64e",
+						Bootable: false,
+						Payload: &disk.Filesystem{
+							Type:         "xfs",
+							Label:        "root",
+							Mountpoint:   "/",
+							FSTabOptions: "defaults",
+							UUID:         "6e4ff95f-f662-45ee-a82a-bdf44a2d0b75",
+							FSTabFreq:    0,
+							FSTabPassNo:  0,
+						},
+					},
+					{
+						Start:    202 * datasizes.MiB,
+						Size:     20 * datasizes.MiB,
+						Type:     disk.FilesystemDataGUID,
+						UUID:     "e2d3d0d0-de6b-48f9-b44c-e85ff044c6b1",
+						Bootable: false,
+						Payload: &disk.Filesystem{
+							Type:         "ext4",
+							Label:        "home",
+							Mountpoint:   "/home",
+							FSTabOptions: "defaults",
+							UUID:         "fb180daf-48a7-4ee0-b10d-394651850fd4",
+							FSTabFreq:    0,
+							FSTabPassNo:  0,
+						},
+					},
+				},
+			},
+		},
+		"lvm": {
+			customizations: &blueprint.DiskCustomization{
+				Partitions: []blueprint.PartitionCustomization{
+					{
+						Type:    "lvm",
+						MinSize: 100 * datasizes.MiB,
+						VGCustomization: blueprint.VGCustomization{
+							Name: "testvg",
+							LogicalVolumes: []blueprint.LVCustomization{
+								{
+									Name:    "varloglv",
+									MinSize: 10 * datasizes.MiB,
+									FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+										Mountpoint: "/var/log",
+										Label:      "var-log",
+										FSType:     "xfs",
+									},
+								},
+								{
+									Name:    "rootlv",
+									MinSize: 50 * datasizes.MiB,
+									FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+										Mountpoint: "/",
+										Label:      "root",
+										FSType:     "xfs",
+									},
+								},
+								{ // unnamed + untyped logical volume
+									MinSize: 100 * datasizes.MiB,
+									FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+										Mountpoint: "/data",
+										Label:      "data",
+										FSType:     "ext4", // TODO: remove when we reintroduce the default fs
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			options: &disk.CustomPartitionTableOptions{
+				DefaultFSType: disk.FS_EXT4,
+				BootMode:      platform.BOOT_HYBRID,
+			},
+			expected: &disk.PartitionTable{
+				Type: disk.PT_GPT, // default when unspecified
+				UUID: "0194fdc2-fa2f-4cc0-81d3-ff12045b73c8",
+				Size: 714*datasizes.MiB + 168*datasizes.MiB + datasizes.MiB, // start + size of last partition (VG) + footer
```

Comment:
(nitpick) It becomes a bit tricky to follow where the 714 comes from, the comment seems to be a bit terse, so maybe a comment like // start + efi + boot + 3 LVs ` or something might slightly more helpful (same below)